ogc/style-wms-layer/src/main/java/com/esri/samples/style_wms_layer/StyleWmsLayerSample.java

```diff
@@ -55,8 +55,10 @@ public void start(Stage stage) {
       stage.setScene(scene);
       stage.show();
 
-      // create a map and add it to the map view
-      ArcGISMap map = new ArcGISMap(Basemap.createImagery());
+      // create a map with spatial reference appropriate for the service
```

Comment:
since these spatial reference id's can be quite obscure, I like to give a little more context in the comments:
```suggestion
      // create a map with spatial reference appropriate for the service (North American Datum 83)
```